src/main/resources/sessionLinksRecoveryEmailTemplate-emailNotFound.html

```diff
@@ -0,0 +1,20 @@
+<p>Hello,</p>
+
+<p>
+    You are receiving this email because an access link recovery request for feedback sessions related to this email address ${userEmail} was made on <a href="${teammateHomePageLink}">TEAMMATES</a>.
+</p>
+
+<p>
+    We could not recover any link as there is no feedback session associated to this email address: ${userEmail}.
+</p>
+
+<p>
+    You can try <a href=${sessionsRecoveryLink}>using a different email address</a> or contact your course instructor(s) to check whether this email address is the one used for your course enrollment on TEAMMATES.</p>
```

Comment:
`is the one used for your course enrollment on TEAMMATES`
-> `is the one used to enroll you in TEAMMATES`?